routes/benchmarks.tsx

```diff
@@ -186,7 +186,8 @@ export default function Benchmarks({ url, data }: PageProps<Data>) {
               >
                 {!showAll ? "all" : "the most recent"}
               </a>{" "}
-              commits.
+              commits or append ?-n to this page's URL to view results for the
```

Comment:
Could you give an example here so it could be copy-pasted? `?-100` would work

Reply:
Done!